perf/perf_watch_point.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Shirisha <shiganta@in..ibm.com>
+
+import os
+import platform
+import shutil
+from avocado import Test
+from avocado import main
+from avocado import skipUnless
+from avocado.utils import cpu, build, distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+CPU_ARCH = cpu.get_cpu_arch().lower()
+IS_POWER8 = 'power8' in CPU_ARCH
```

Comment:
should  be like 

`IS_POWER8 = 'power8' in cpu.get_cpu_arch().lower()` as  minimize use of extra variable where ever possible

Reply:
yeah sure